Server/Test_Plan_OCIS_Compatibility_Test.md

```diff
@@ -0,0 +1,34 @@
+## Compatibility test:
+Note: It is necessary to switch to the latest stable version and do all the steps below.
+After switching to new ocis version you need check that all resources, shares, settings still exist and remain unchanged
+
+
+### from version 1 to version 2
```

Comment:
What combinations do we usually want to check?

For example, there is now ocis major version 1, 2 and 3. And let's suppose that there are minor versions 3.1 and 3.2 released.

When the next major version 4 is released, then what upgrade paths do we check:
1) start with a working 3.2 installation, upgrade to 4.0 and test
2) start with a working 3.1 installation, upgrade to 4.0 and test
3) start with a working 3.0 installation, upgrade to 4.0 and test
4) start with a working 2.* installation, upgrade to 4.0 and test
5) start with a working 1.* installation, upgrade to 4.0 and test

I suppose that we do not directly support (4) or (5). Installations should upgrade one major version at a time. So before going to v4 they should first get from v1 to v2 to v3, step by step. So we only need to test upgrade from v3 to v4.

Do we expect that installations will first upgrade to the latest minor release (e.g. 3.2 in this example) and then to v4? If so, then we only need to test item (3).

Reply:
> Do we expect that installations will first upgrade to the latest minor release (e.g. 3.2 in this example) and then to v4? If so, then we only need to test item (1).

Yes, this is what happens when testing a release

Reply:
It can be test between 3.1 and 3.2. the latest release with a new release